src/Compilers/CSharp/Portable/Symbols/Wrapped/WrappedNamedTypeSymbol.cs

```diff
@@ -95,7 +95,7 @@ internal override bool MangleName
             }
         }
 
-        internal override SyntaxTree AssociatedSyntaxTree => _underlyingType.AssociatedSyntaxTree;
+        internal override FileIdentifier? AssociatedFileIdentifier => _underlyingType.AssociatedFileIdentifier;
```

Comment:
>_underlyingType.AssociatedFileIdentifier [](http://example.com/codeflow?start=70&length=40)

Why is this a good default? Perhaps the override should be abstract?

Reply:
I pushed this implementation down into types where it is more clear that this is the correct implementation. In some cases I changed the implementation to just return null because it did not seem to make sense to return a real file identifier.

I wasn't 100% sure whether it is possible in EE to refer to a retargeting symbol whose underlying symbol is file-local.